umbraco-cloud/set-up/project-settings/availability-performance.md

```diff
@@ -102,6 +103,40 @@ Cloud projects on shared resources with a granularity of 5 minutes, will see the
 
 For cloud projects with a dedicated option (or shared plans with another granularity than 5 minutes), users will see the average assigned private bytes in bytes. Here the statistics panel will display the maximum, average, and minimum allocation of private bytes based on selected granularity.
 
+#### Platform and CMS events
+
+The charts are enhanced with platform event like restarts, automatic and manual upgrades, environment-to-environment deployments and plan changes.
+
+This information will assist you in potential troubleshooting, make informed decisions, and ensure smooth project management.
+
+By utilizing the `Umbraco.Cloud.Cms` package we are tracking the hot and colds boots of your Umbraco environment in cloud.
+The package is installed on v10, v13 and v14 environments in Umbraco Cloud.
```

Comment:
Can we future proof this line? make itsomething like on v10, v13, v14 and all new and upcoming versions

Reply:
I've changed it a bit + and moved one bit around - u like?

Reply:
How about:
"The package is installed on all new and upcoming versions of Umbraco, incl. versions 10, 13, and 14."